pfcwd/main.py

```diff
@@ -92,7 +92,8 @@ def config(ports):
             line = config_entry.get(config[1], config[2])
             config_list.append(line)
         table.append([port] + config_list)
-
+    poll_interval = configdb.get_entry( 'PFC_WD_TABLE', 'GLOBAL').get('POLL_INTERVAL')
```

Comment:
Does someone write default value there or should we check if it's not empty?

Reply:
changed to display only when the value is changed.